### PR TITLE
fix main.css link in params.html

### PR DESCRIPTION
**Pull request by jiayliu**

### CODE REVIEW

**Comment by chuckhays** [comment]
LGTM.

File: src/web_app/html/params.html [hunk 1]
```diff
@@ -89,8 +89,5 @@
     <a href="//github.com/GoogleChrome/webrtc" title="View the repository" id="viewSource">github.com/GoogleChrome/webrtc</a>
 
   </div>
-
```
Comment (andresusanopinto) [inline src/web_app/html/params.html]: @jiayliu 
Was this intentional?
Why is it missing from the pull request description?

Comment (jiayliu) [inline src/web_app/html/params.html]: Oops, I didn't realize it pointed to the ga.js in samples. Will add it back.

Comment (jiayliu) [inline src/web_app/html/params.html]: But wait, why do we need it? This is page is not in samples. @andresusanopinto

Comment (andresusanopinto) [inline src/web_app/html/params.html]: I have no clue if it is needed or not, and did not spend any brain on it.. i just happen to be browsing changes (when apprtc was not working) and though it was strange a fix for .css path touched this line.

**Pull request merged**